src/test/java/com/artipie/rpm/RpmTest.java

```diff
@@ -132,6 +139,29 @@ void updatesDifferentReposSimultaneouslyTwice(final UpdateType update) throws Ex
         );
     }
 
+    @ParameterizedTest
+    @EnumSource(UpdateType.class)
+    @Order(Integer.MAX_VALUE)
```

Comment:
@olenagerasimova could you please explain why order is important in this test suite? Usually it's a bad sign meaning tests are too coupled

Reply:
@olegmoz on each update `Rpm` create two temp dirs with prefixed `meta` and `repo`, running this test after all other tests we can be sure, that not only temp dirs from this particular update were removed, but also all temp dirs from each update of this test class. I've caught the error with `onTerminate` this way... 
I understand, that this logic is not obvious and honestly I'm not a fun of this approach. Another solution I can think of here is to add these assertions to `@After` method, but it does not seem to be a nice way either. What do you say?

Reply:
@olenagerasimova adding the checks to `@After` method does not sound good. I guess proper way to do it is to add a check to every test case